src/backoffice/settings/languages/workspace/language/language-workspace-edit.element.ts

```diff
@@ -58,19 +78,28 @@ export class UmbLanguageWorkspaceThingyElement extends UmbLitElement {
 	render() {
 		return html`<umb-workspace-layout alias="Umb.Workspace.Language">
 			<div id="header" slot="header">
-				<uui-button href="/section/settings/workspace/language-root" compact>
+				<uui-button label="Navigate back" href="/section/settings/workspace/language-root" compact>
```

Comment:
I know this isn't new, but links should be relative because we are using a `<base>` tag

```suggestion
				<uui-button label="Navigate back" href="section/settings/workspace/language-root" compact>
```